src/Modules/CalcPerform.lua

```diff
@@ -1729,7 +1742,7 @@ function calcs.perform(env, avoidCache)
 					local curse = {
 						name = buff.name,
 						fromPlayer = true,
-						priority = activeSkill.skillTypes[SkillType.Aura] and 3 or 1,
+						priority = determineCursePriority(buff.name, activeSkill.socketGroup.slot, activeSkill.socketGroup.source) + (activeSkill.skillTypes[SkillType.Aura] and 30000 or 10000),
```

Comment:
Can we change these magic numbers to constants or at least calculate them as part of the `determineCursePriority` function?